jdaviz/configs/cubeviz/helper.py

```diff
@@ -38,7 +38,8 @@ def _set_spectrum_x_axis(self, msg):
                         viewer.state.reset_limits()
                     break
             else:
-                viewer.state.x_att_pixel = ref_data.id["Pixel Axis 2 [x]"]
+                viewer.state.x_att = ref_data.id["Pixel Axis 2 [x]"]
+                viewer.state.reset_limits()
```

Comment:
why was this needed to be added?

Reply:
To be consistent with similar logic above:

https://github.com/spacetelescope/jdaviz/blob/a7837c447c9eab53ae4b65810c1e61115ab81e8c/jdaviz/configs/cubeviz/helper.py#L37-L38